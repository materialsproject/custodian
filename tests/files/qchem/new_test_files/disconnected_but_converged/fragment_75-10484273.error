/global/u2/s/sblau/pymatgen/pymatgen/io/cif.py:44: UserWarning: Please install optional dependency pybtex if youwant to extract references from CIF files.
  warnings.warn("Please install optional dependency pybtex if you"
DEBUG:launchpad:FW with id: 3553 is unique!
DEBUG:launchpad:Created/updated Launch with launch_id: 5960
DEBUG:launchpad:RUNNING FW with id: 3553
INFO:rocket.launcher:RUNNING fw_id: 3553 in directory: /global/u2/s/sblau/block_2018-08-23-17-39-12-141126/launcher_2018-08-24-19-58-10-170106
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
INFO:rocket.launcher:Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
INFO:custodian.custodian:Run started at 2018-08-24 12:58:29.761650 in /global/u2/s/sblau/block_2018-08-23-17-39-12-141126/launcher_2018-08-24-19-58-10-170106.
INFO:custodian.custodian:Custodian running on Python version 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 18:10:19)  [GCC 7.2.0]
INFO:custodian.custodian:Hostname: nid02549, Cluster: unknown
INFO:custodian.custodian:Starting job no. 1 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 2 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Logging to custodian.json...
INFO:custodian.custodian:Run ended at 2018-08-24 13:06:51.409560.
INFO:custodian.custodian:Run completed. Total time taken = 0:08:21.647910.
