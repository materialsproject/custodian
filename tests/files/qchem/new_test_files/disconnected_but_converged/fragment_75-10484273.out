foobar
2018-08-24 12:58:22,461 INFO Hostname/IP lookup (this will take a few seconds)
2018-08-24 12:58:22,464 INFO Launching Rocket
2018-08-24 12:58:29,661 INFO RUNNING fw_id: 3553 in directory: /global/u2/s/sblau/block_2018-08-23-17-39-12-141126/launcher_2018-08-24-19-58-10-170106
2018-08-24 12:58:29,704 INFO Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
2018-08-24 12:58:29,726 INFO Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
2018-08-24 12:58:29,761 INFO Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
